recipe/build.sh

```diff
@@ -47,6 +47,18 @@ if [[ ("${target_platform}" == "win-64" && "${build_platform}" == "linux-64") ]]
   # maturin will expect libpython DSOs at PYO3_CROSS_LIB_DIR
   # which we don't have since we are not able to add python as a host dependency
   cargo feature pyo3 +generate-import-lib --manifest-path py-polars/Cargo.toml
+
+  # cc-rs hardcodes ml64.exe as the MASM assembler for x86_64-pc-windows-msvc
+  # We want to use LLVM's MASM assembler instead
+  # https://github.com/rust-lang/cc-rs/issues/1022
+  mkdir -p $PREFIX/bin
+  cat > $PREFIX/bin/ml64.exe <<"EOF"
```

Comment:
I think this will result in ml64.exe being Part of our conda package 🤔 either remove this file afterwards or put it somewhere else on PATH (like `$BUILD_PREFIX/bin`)

Reply:
I changed `PREFIX` -> `BUILD_PREFIX` now.